src/miner.cpp

```diff
@@ -917,22 +917,22 @@ void ThreadStaker::operator()(std::vector<ThreadStaker::Args> args, CChainParams
             }
             static std::atomic<uint64_t> time{0};
             if (GetSystemTimeInSeconds() - time > 120) {
-                LogPrintf("ThreadStaker: unlock wallet to start minting...\n");
+                LogPrint(BCLog::STAKING, "ThreadStaker: unlock wallet to start minting...\n");
                 time = GetSystemTimeInSeconds();
             }
             std::this_thread::sleep_for(std::chrono::milliseconds(100));
         }
     }
 
-    LogPrintf("ThreadStaker: started.\n");
+    LogPrint(BCLog::STAKING, "ThreadStaker: started.\n");
 
     while (!args.empty()) {
         boost::this_thread::interruption_point();
 
         while (fImporting || fReindex) {
             boost::this_thread::interruption_point();
 
-            LogPrintf("ThreadStaker: waiting reindex...\n");
+            LogPrint(BCLog::STAKING, "ThreadStaker: waiting reindex...\n");
```

Comment:
A better approach here would be use a throttled logger method for staking log, that throttles psuedo actionable info, like `waiting reindex`, `waiting init`.